Loader/Loader/Program.cs

```diff
@@ -6,70 +6,92 @@ namespace Loader
 {
     class Program
     {
-        // public static bool RunChellenge(Action<StreamReader, StreamWriter> challenge, string inputPath, string outputPath)
-        public static bool RunChellenge(int c, int d)
+        // public static bool RunChallenge(Action<StreamReader, StreamWriter> challenge, string inputPath, string outputPath)
+        public static bool RunChallenge(int c, int d)
         {
-            ////////////////////проверка существования
+            //check for file existence
             string path = @"C:\Users\Даша\source\repos\tmp.txt";
-            if (Directory.Exists(path))
+            if (File.Exists(path))
             {
-                Console.WriteLine("File is not found");
-            }
+                string text1 ="";
```

Comment:
1. For empty strings it's better to use the predefined constant "string.Empty"
2. If any variable assigns in further you'd better leave it unassigned (just "string text1;")
3. Try omit usage of such variable names as "a, b, text1, text2" etc. if C#. Do not afraid of long names also.
Try to name it that way so it will be clear what it stands for.